.prettierrc.json

```diff
@@ -0,0 +1,5 @@
+{
+    "trailingComma": "es5",
+    "tabWidth": 4,
+    "printWidth": 120 
```

Comment:
The software that will enforce trailing commas can't parse it's config file if it has them.

Reply:
hah